sktime/datatypes/_registry.py

```diff
@@ -51,32 +58,38 @@
     "MTYPE_LIST_PANEL",
     "MTYPE_LIST_SERIES",
     "SCITYPE_REGISTER",
+    "Scitype",
 ]
 
 
-SCITYPE_REGISTER = [
-    ("Series", "uni- or multivariate time series"),
-    ("Panel", "panel of uni- or multivariate time series"),
-]
+class Scitype(BaseRegistryEnum):
+    """Enum class for scitypes."""
+
+    SERIES = ("Series", "uni- or multivariate time series")
+    PANEL = ("Panel", "panel of uni- or multivariate time series")
 
 
-def mtype_to_scitype(mtype: str):
+SCITYPE_REGISTER = [tuple(scitype) for scitype in Scitype]
+
+
+def mtype_to_scitype(mtype: Union[PanelMtype, SeriesMtype, str]) -> str:
```

Comment:
should this not just be `Mtype`?

Reply:
Unfortunatly I can't import the types into this due to circular importing.

Reply:
Argh - is there any way to resolve this, e.g., by splitting files?

That is, is the circularity coming through object imports, or just via module imports?

Reply:
Module imports since the types are based off the enums defined in that file. I can move it but it was where you orginally defined mtype_to_scitpye where would you want me to put it instead?

Reply:
really good question - perhaps the enums should move somewhere, e.g., `types`, which contains the enums and any logic that allows shorthand use as a `typing` type?